docs/general-development/how-to-create-sharepoint-workflows-using-visual-studio.md

```diff
@@ -188,7 +189,7 @@ To create a custom workflow using Visual Studio 2012 or later, ensure that you h
 5. When prompted by the **SharePoint Customization Wizard** for a name, leave the default, then set it to be a **List Workflow**. Click **Next**.
     
   
-6. On the next page of the wizard, check the box to create an association, then select the **Announcements** list that we just created; select **<Create New>** for the required workflow history and task lists and then click **Next**.
+6. On the next page of the wizard, check the box to create an association, then select the **Announcements** list that we just created; select **&lt;Create New&gt;** for the required workflow history and task lists and then click **Next**.

```

Comment:
Correct markdown escaping of angled brackets should be prefixed with`\`; don't change to the HTML encoding.